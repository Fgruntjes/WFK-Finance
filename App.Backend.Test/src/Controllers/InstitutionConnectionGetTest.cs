namespace App.Backend.Test.Controllers;

public class InstitutionConnectionGetTest : IClassFixture<InstitutionConnectionGetFixture>
{
	private readonly InstitutionConnectionGetFixture _fixture;

	public InstitutionConnectionGetTest(InstitutionConnectionGetFixture fixture)
	{
		_fixture = fixture;
	}

	[Fact]
	public async Task Success()
	{
		var result = await _fixture.ExecuteQuery(new { Id = _fixture.OrganisationMatchConnection });
		result.MatchSnapshot();
	}

	[Fact]
	public async Task OrganisationMismatch()
	{
		var result = await _fixture.ExecuteQuery(new { Id = _fixture.OrganisationMissmatchConnection });
		result.MatchSnapshot();
	}

	[Fact]
	public async Task WithInstitution()
	{
		var result = await _fixture.ExecuteQuery(new { Id = _fixture.OrganisationMatchConnection });
		result.MatchSnapshot();
	}

	[Fact]
	public async Task WithAccounts()
	{
		var result = await _fixture.ExecuteQuery(new { Id = _fixture.OrganisationMatchConnection });
		result.MatchSnapshot();
	}
}