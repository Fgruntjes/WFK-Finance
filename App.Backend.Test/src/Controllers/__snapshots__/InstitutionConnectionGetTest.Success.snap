{
  "data": {
    "institutionConnection": {
      "get": {
        "externalId": "SomeExternalId-organisation-match-0"
      }
    }
  }
}
